2023/naughtylist.pod

```diff
@@ -0,0 +1,190 @@
+Title: Santa's Christmas Dancer2 Web App - Naughty & Nice List Manager
+Topic: HMLX and Dancer2
+Author: Ken Town <ktown007@gmail.com>
+
+=encoding utf8
+
+=pod 
+
+Ho, ho, ho! In this hilarious Christmas-themed Dancer2 web app tutorial, we'll help Santa manage his Naughty & Nice list. This time, we're using Perl's Dancer2 framework, but Santa's Christmas spirit is universal. Let's dive in and create Santa's very own web app!
+
+
+=head2 todo.pl
+
+    !/usr/bin/env perl
+
+    use Dancer2;
+    set template => 'mustache';
+
+        #Initialize Santa's Naughty & Nice List
+
+    my $list = { list => [] };
```

Comment:
Maybe persist the list?

Reply:
I had the thought to do a part 2 using planetscale db and a part 3 with a fly.io deployment. Details of the final version are here:
https://github.com/ktown007/todoapp
if you think there is interest expanding on the array of hash to table of records concept could be done. "Santa's favorite Dancer Fly.io(s) to PlanetScale in time for Christmas"...

Reply:
This was also meant to contrast with all the other to do app tutorials for front end frameworks. In keeping it simple I was surprised how easy it was.